cpp/cmake/Modules/ConfigureNvgraph.cmake

```diff
@@ -20,7 +20,7 @@ endif(NOT CMAKE_CXX11_ABI)
 file(MAKE_DIRECTORY "${NVGRAPH_ROOT}/cpp/build")
 #file(MAKE_DIRECTORY "${NVGRAPH_ROOT}/install")
 
-execute_process(COMMAND ${CMAKE_COMMAND} -G ${CMAKE_GENERATOR} .. -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX} -DNVGRAPH_LIGHT=${NVGRAPH_LIGHT}
+execute_process(COMMAND ${CMAKE_COMMAND} -G ${CMAKE_GENERATOR} .. -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX} -DNVGRAPH_LIGHT=${NVGRAPH_LIGHT}
```

Comment:
I don't see `include(ConfigureNvgraph)` being called in the main cmakelists. Should we delete this file?

Reply:
Nice catch, and I will delete the file.